viewing_party/party.py

```diff
@@ -1,23 +1,177 @@
+import json
+from textwrap import indent
+
 # ------------- WAVE 1 --------------------
 
+# Instantiate movie as dictionary if all fields are populated
 def create_movie(title, genre, rating):
-    pass
+    if title and genre and rating:
+        movie = {
+            "title" : title,
+            "genre" : genre,
+            "rating" : rating,
+        }
+        return movie
+
+# User_data set up as dictonary with the key watched corresponding to a list of movie dictionaries

```

Comment:
I like these comments! Consider converting them to docstring format with python multiline comments: ` ``` `
Not only do they help with formatting (not as many #s) but IDEs have support for displaying them as well as automatically generating documentation.